addons/web/static/src/views/fields/progress_bar/progress_bar_field.js

```diff
@@ -150,10 +156,13 @@ ProgressBarField.displayName = _lt("Progress Bar");
 ProgressBarField.supportedTypes = ["integer", "float"];
 
 ProgressBarField.extractProps = ({ attrs }) => {
+    if (!isNaN(attrs.options.max_value)) {
+        browser.console.warn(`${attrs.options.max_value} cannot be set as max_value. You must use the name of another field in the view.`);
+    }
     return {
         maxValueField: attrs.options.max_value,
         currentValueField: attrs.options.current_value,
-        isPercentage: !attrs.options.max_value,
+        isPercentage: !attrs.options.max_value || !isNaN(attrs.options.max_value),
```

Comment:
I know it's not directly related to this PR but, should this be a props ? It can be derived from another props

Reply:
You're right. Done